manifests/server.pp

```diff
@@ -16,6 +16,7 @@
     $ping             = false,
     $ping_restart     = false,
     $mtu_discovery    = true,
+    $server_mode      = 'normal',
```

Comment:
What does `'normal'` mode mean? What are the alternate modes? Could the class accept values for the `ifconfig` and `server-bridge` parameters instead?